zebROS_ws/src/behaviors/src/game_piece_path_gen.cpp

```diff
@@ -0,0 +1,137 @@
+#include <ros/ros.h>
```

Comment:
Just an FYI here, if you use `git mv` rather than just `mv`, it will help git figure out that you're renaming a file vs. removing the old one and creating a new one.  It can sometimes figure it out anyway, but in cases like this where there are lots of changes it'll miss it.
No need to fix anything here,  just something to keep in mind next time.